iu_xray_ontology.json

```diff
@@ -0,0 +1,53 @@
+
+{
+    "name": "iu_xray_ontology",
+    "additional_prefixes": [
+      "iu_xray.onto.iu_xray_ontology"
+    ],
+    "definitions": [
+      {
+        "entry_name": "iu_xray.onto.iu_xray_ontology.Findings",
```

Comment:
Can you following this example and use `Annotation` instead of `Generic`?

https://github.com/petuum/medical_images/blob/main/mimic_ontology.json#L9

Reply:
modified